README.md

```diff
@@ -1,13 +1,16 @@
 # Project Todos
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This is a to do application that uses local storage to save your very own to do list. There are two tasks already created at the start to show the design of the posts, but thanks to local storage, once you delete them they won't come back. It's also possible to add new to dos as well as delete created ones. You also get a date stamp when the to do was created. When you check a to do as complete it will be sorted to the end of the list of all tasks.
```

Comment:
Complete and detailed ReadMe that leaves no doubts to the reviewer regarding the project .